schemas/org/openstax/ec/types/session.rb

```diff
@@ -0,0 +1,3 @@
+record :session do
+  required :session_uuid, :uuid
```

Comment:
This type will only ever be used in the `started_session` event, so I don't know how useful it is to pull it out like this.

Reply:
was doing this just to create consistency of this abstraction.